src/api/procedures/modifyToken.ts

```diff
@@ -88,6 +108,22 @@ export async function prepareModifyToken(
     );
   }
 
+  if (newIdentifiers) {
+    if (isEqual(identifiers, newIdentifiers)) {
```

Comment:
This is not enough to compare arrays. If the arrays have the same elements but in a different order, this check would be false when it should be true.

To check if an array `a` has the same elements as an array `b`, you have to check `if (!differenceWith(a, b, isEqual).length && a.length === b.length)` (`differenceWith` and `isEqual` from `lodash`)